src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3670/CreateTransactionEOF1ValidOpcodesFiller.yml

```diff
@@ -0,0 +1,1702 @@
+# Creation transaction deploying EOF code containing each valid Opcode
+---
+CreateTransactionEOF1ValidOpcodes:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      # Deployed EOF code: PUSH1(0) DUP1(16 times) <opcode>
```

Comment:
Perhaps add STOP in the end

Reply:
Yes, STOP would be still required, otherwise 5450 validation will fail.

I guess I got confused in previous comment.